app/assets/javascripts/listings/templates/listing/_eligibility.html.slim

```diff
@@ -135,7 +135,7 @@ accordion-heading.lead
               ng-pluralize count="units.length" when="{'0': ' units', '1': ' unit', 'other': ' units'}"
           p.c-steel translate="LISTINGS.UNITS_ARE_FOR" translate-value-type="{{::reservedLabel(listing, type, 'unitDescription')}}"
 
-    li.custom-counter_item ng-if="::listingHasPriorityUnits()"
+    li.custom-counter_item ng-if="listingHasPriorityUnits()"
```

Comment:
@callaghanc I was able to grab some time with chris this afternoon to learn a bit more about the angular data binding for this.

She suggested that there are 2 options for fixing the problem
1. Remove the one-time data binding (`::`) for this ng-if. 
This means that the function is getting called constantly (including on page scroll), but is also means that when the request for listing units completes, the value is updated. One concern with this is that it could negatively impact performance. I couldn't notice a difference with a quick test, but I'm not sure how we should best evaluate the potential performance hit. 
2. Add a boolean variable to the scope for hasPriorityUnits, then update it when the priority units are set. This seems less-ideal because then we are adding another variable to the scope, but has the benefit of not being called constantly because it'll only update once, when the value is set. 

What are your thoughts? We can discuss in a slack call tomorrow.